pkg/southbound/topo/topo.go

```diff
@@ -40,11 +40,11 @@ var log = logging.GetLogger("southbound", "topo")
 func ConnectToTopo(ctx context.Context, topoEndpoint string, serverParams utils.ServerParams) topodevice.DeviceServiceClient {
 	log.Infof("Connecting to ONOS Topo...%s", topoEndpoint)
 	// Attempt to create connection to the Topo
-	opts, err := certs.HandleCertArgs(&serverParams.KeyPath, &serverParams.CertPath)
-	opts = append(opts, grpc.WithStreamInterceptor(southbound.RetryingStreamClientInterceptor(100*time.Millisecond)))
```

Comment:
Was it your intention to remove the retry mechanism here?

Reply:
i just moved it down to after the error check

Reply:
Derp, I didn't see that.